### ISSUE TITLE
500 Internal Server Error Server got itself in trouble

### ISSUE THREAD

**Issue by mansiondeluxe** [nonsleepr/ha-eufy-security#82]
Someone else got this?!

**Comment by Egregius** [comment]
Yes, me... 
Trying to get my Eufy doorbell connected but get this error: Config flow could not be loaded: 500 Internal Server Error Server got itself in trouble

**Comment by Tschaegged** [comment]
Yes, I try to configure the Video Doorbell E340 and get this message

**Comment by lukestein** [comment]
I did too. Is this integration dead?
